### PR TITLE
Develop1

### PR DESCRIPTION
**Pull request by aceburn24**

### CODE REVIEW

File: src/Admin/Dashboard.js [hunk 1]
```diff
@@ -36,14 +45,12 @@ const menu = [
...
         link: "/roles",
+        icon: BsPersonBadgeFill,
       },
       {
         text: "Customers",
         link: "/customers",
-      },
-      {
-        text: "Authentication Settings",
-        link: "/auth",
```
Comment (pjguitar15) [inline src/Admin/Dashboard.js:46]: bakit remove? di na need to?

File: src/Admin/Dashboard.js [hunk 2]
```diff
@@ -72,14 +82,13 @@ const menu = [
       {
         text: "Orders & Tracking",
         link: "/orders",
+        icon: BsWalletFill,
       },
-      {
```
Comment (pjguitar15) [inline src/Admin/Dashboard.js:76]: Same here, if no need na to. feel free to resolve. just making sure na ok iremove to

**Review by pjguitar15 — APPROVED** [review]

File: src/Admin/Dashboard.js (hunk 1 above)
Comment (aceburn24) [reply src/Admin/Dashboard.js:46]: di na

File: src/Admin/Dashboard.js (hunk 2 above)
Comment (aceburn24) [reply src/Admin/Dashboard.js:76]: yes

**Pull request merged**